web/src/beta/components/fields/LocationField/index.stories.tsx

```diff
@@ -0,0 +1,18 @@
+import { action } from "@storybook/addon-actions";
+import { Meta, StoryObj } from "@storybook/react";
+
+import LocationField from ".";
+
+const meta: Meta<typeof LocationField> = {
+  component: LocationField,
+};
+
+export default meta;
+
+type Story = StoryObj<typeof LocationField>;
+
+export const LocationFieldUI: Story = {
+  render: () => (
+    <LocationField name="Location" description="The Location field" onChange={action("onchange")} />
+  ),
+};
```

Comment:
```suggestion
export const Sample: Story = {
  args: {
    name: "Location Field",
    description: "Location field description",
    value: { lat: 87.43, lng: 107.53 },
    onChange: () => console.log("clicked"),
  },
};
```

If you write it like this, the props can be changed in the stories UI

Reply:
![image](https://github.com/reearth/reearth/assets/20891087/58a0952a-8d56-4e75-b1ce-c93480a11d0d)